Controllers/IncomingSms.cs

```diff
@@ -29,10 +30,13 @@ public class IncomingSmsController : ControllerBase
     public IncomingSmsController(ILogger<IncomingSmsController> logger)
     {
         _logger = logger;
-
-        string accountId = System.Environment.GetEnvironmentVariable("ACCOUNT_ID");
-        string apiKey = System.Environment.GetEnvironmentVariable("API_KEY");
+        DotNetEnv.Env.Load();
+        
+        string apiServer = System.Environment.GetEnvironmentVariable("API_SERVER") ?? "https://www.freeclimb.com/apiserver";
+        string? accountId = System.Environment.GetEnvironmentVariable("ACCOUNT_ID");
```

Comment:
Do we need a question mark in front if we have a check that it should be supplied?